roles/openshift_hosted_templates/files/v3.6/origin/registry-console.yaml

```diff
@@ -89,7 +89,7 @@ objects:
         - annotations: null
           from:
             kind: DockerImage
-            name: ${IMAGE_NAME}:${IMAGE_VERSION}
+            name: ${IMAGE_PREFIX}kubernetes:${IMAGE_VERSION}
```

Comment:
why is kubernetes added here now?

Reply:
It is consistent with how the enterprise image is constructed (where `registry-console` is hardcoded) and indeed how every other image in constructed (from a prefix and a partial name).